### DISCUSSION TITLE
Merge faces sorting by similarity doesn't help

### DISCUSSION THREAD

**Discussion by swingtempo** [immich-app/immich#14762] [Feature Request]
### The bug

I'd like the option to sort my similarity _or_ frequency. Some of the baby pictures I'm trying to disambiguate are of kids that are very frequent, and the face detection is matching them to the wrong face. I'd like the option to go back to the frequency sort order so I don't have to type the same kid name each time :)

### The OS that Immich Server is running on

Windows Docker on WSL2

### Version of Immich Server

1.123.0

### Version of Immich Mobile App

n/a

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
n/a
```

### Your .env content

```Shell
n/a
```

### Reproduction steps

1. See a baby face that is matched to wrong face.
2. Try to adjust
3. Preference: let me go back to the original sort order instead of the current face-similarity order since 99% it is one of the top 5 kids.

**Comment by mertalev** [comment]
Thanks for the feedback! I converted this to a discussion since this isn’t really a bug.

**Reply by swingtempo** [reply]
Thank you. Would love to get the old behavior as an option.  Sometimes I want similarity, sometimes not :)

**Comment by bo0tzz** [comment]
#14830
